scoped-transitions.md

```diff
@@ -0,0 +1,110 @@
+
+3. The captured output above is painted in a stacking context which is a descendant of scoped-transition-root's stacking context. The above example generates the following stacking context hierarchy.
+
+   ```
+   root
+   └─ outer
+      ├─ inner
+      └─ outer-page-transition (pseudo-tree underneath)
+   ```
+
+   If the scoped-transition-root is tagged, it's snapshot uses [content-capture](https://github.com/WICG/shared-element-transitions/blob/main/explainer.md#more-granular-style-capture) mode. This means that only the element's descendants (including both text and elements) are painted in the snapshot while box decorations (like border, box-shadow) continue to paint in the outer stacking context.
+
+4. Once the `updateDOM` callback is finished, incoming-image pseudo-elements are added to the tree generated in step 2. Default animations are added as defined in [7.8](https://drafts.csswg.org/css-shared-element-transitions-1/#animate-a-page-transition-algorithm).
+
+## Constraints
+
+An element which is the root for a transition has the following constraints. These constraints may stop being satisfied at any phase of the transition lifecycle and we'll need to define the fallback behavior for such cases. The simplest option is to consider them a developer error and [skip the transition](https://drafts.csswg.org/css-shared-element-transitions-1/#skip-the-page-transition).
+
+1. There shouldn't be an active transition on the element, its descendants or its ancestors. This is because a transition involves taking the painted output of an element and rendering it into a pseudo-element. For example, in the following case:
+
+   ```html
+   <style>
+     .outer {
+       page-transition-tag: outer;
+     }
+     .inner {
+       page-transition-tag: inner;
+     }
+   </style>
+   <div class="outer">
+     <div class="inner"></div>
+   </div>
+   ```
+
+   ```js
+   document.createTransition(…);
+   document.querySelector('.outer').createTransition(…);
+   ```
+
+   There will be an image pseudo-element for `inner` in transitions on document and `outer` and its ambiguous which pseudo-element it should be painted in.
+
+2. The element which is the root for a transition must have `contain: layout`. This ensures it generates a stacking context and its painted output can be captured as an atomic unit.
+
+## Selector Syntax
+
+The CSS selector options for the pseudo-trees can be:
+
+- [part](https://w3c.github.io/csswg-drafts/css-shadow-parts-1/#part) selector which is spec'd to match elements in shadow DOM originating from an element. `html::part(container)` for document transitions and `#outer::part(container)` if outer is the scoped-transition-root. This implies that `#outer` can not be a shadow host for author's shadow DOM since which shadow DOM `#outer::part(container)` is targeting is ambiguous.
```

Comment:
Yeah this seems bad 😄

Reply:
I left a comment on your doc to cover this there.